cpp/src/centrality/betweenness_centrality_kernels.cuh

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+namespace cugraph {
+namespace detail {
+// Dependecy Accumulation: based on McLaughlin and Bader, 2018
+// FIXME: Accumulation kernel mights not scale well, as each thread is handling
+//        all the edges for each node, an approach similar to the traversal
```

Comment:
Do I remember correctly that this is still a small fraction of the total time on large cases?

Reply:
In most of our tests the accumulation kernels represents roughly 25% of our kernels time.
However in our worst test case the accumulation kernel turns out to be the bottleneck by being close to 90%.